frontend/views/Overview.tsx

```diff
@@ -159,6 +159,7 @@ const Overview = () => {
             width: 380,
```

Comment:
```suggestion
            width: 400,
```

Reply:
we could increase to 400 and set the same width as you suggested for the `.MuiDrawer-paper`